src/test/java/jp/co/yahoo/yosegi/spread/column/TestPrimitiveCellManager.java

```diff
@@ -0,0 +1,346 @@
+    assertNull( cellManager.get( 1 , null ) );
+    assertNull( cellManager.get( 2 , null ) );
+
+    ICell cell0 = cellManager.get( 3 , null );
+    assertEquals( ( (PrimitiveObject)( cell0.getRow() ) ).getString() , "test" );
+  }
+
+  @Test
+  public void T_add_get_4() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 3 );
+
+    assertThrows( RuntimeException.class ,
+      () -> {
+        cellManager.add( new StringObj( "test" ) , 3 );
+      }
+    );
+  }
+
+  @Test
+  public void T_add_get_5() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 3 );
+
+    assertThrows( RuntimeException.class ,
+      () -> {
+        cellManager.add( new StringObj( "test" ) , 1 );
+      }
+    );
+  }
+
+  @Test
+  public void T_add_get_6() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+
+    assertThrows( RuntimeException.class ,
+      () -> {
+        cellManager.add( new StringObj( "test" ) , -1 );
+      }
+    );
+  }
+
+  @Test
+  public void T_add_get_7() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 0 );
+    cellManager.add( new StringObj( "test1" ) , 5 );
+
+    ICell cell0 = cellManager.get( 0 , null );
+    assertEquals( ( (PrimitiveObject)( cell0.getRow() ) ).getString() , "test" );
+
+    assertNull( cellManager.get( 1 , null ) );
+    assertNull( cellManager.get( 2 , null ) );
+    assertNull( cellManager.get( 3 , null ) );
+    assertNull( cellManager.get( 4 , null ) );
+    ICell cell1 = cellManager.get( 5 , null );
+    assertEquals( ( (PrimitiveObject)( cell1.getRow() ) ).getString() , "test1" );
+  }
+
+  @Test
+  public void T_add_get_8() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 3 );
+    cellManager.add( new StringObj( "test1" ) , 5 );
+
+    assertNull( cellManager.get( 0 , null ) );
+    assertNull( cellManager.get( 1 , null ) );
+    assertNull( cellManager.get( 2 , null ) );
+
+    ICell cell0 = cellManager.get( 3 , null );
+    assertEquals( ( (PrimitiveObject)( cell0.getRow() ) ).getString() , "test" );
+
+    assertNull( cellManager.get( 4 , null ) );
+    ICell cell1 = cellManager.get( 5 , null );
+    assertEquals( ( (PrimitiveObject)( cell1.getRow() ) ).getString() , "test1" );
+  }
+
```

Comment:
Please describe what are tested in the test method separately at T_add_get_?.
For example, T_add_get_when_xxx or some.

Reply:
Please add a prefix or suffix like T_ or Text or some so that the code reader knows it is a test method including assertThat or assertThrows and/or some testing methods.